util/pgo/basic_pgo_new.sh

```
@@ -19,6 +19,22 @@ function check() {
     fi
 }
 
+function warn() {
+    echo "WARN: $1"
+}
+
+function rebuild_without_pgo() {
+    warn "$1"
+    cd "$gem5_home"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)